### DISCUSSION TITLE
Can i connect without check token?

### DISCUSSION THREAD

**Discussion by DoGunKIM93** [TabbyML/tabby#2874] [General]
Thank you for the good project.

Are there any options to skip check the token?
Sometimes I can't access the web after running tabbyml. 
So I can't get the token to input my vscode extension.

**Comment by wsxiaoys** [comment]
If you don't have access to the web, how will you be able to access Tabby's server?

Nonetheless, you can set the token once, and it should remain valid until you manually refresh it.

**Reply by DoGunKIM93** [reply]
Thank you for your reply.

a network connection is ok. but I can't access open the browser using localhost:8080.
out platform does not approve this access. 
so where can i set server side static token? or how to get the token as api call?
